scripts/style.sh

```diff
@@ -21,19 +21,61 @@
 # Commonly
 # ./scripts/style.sh master
 
-if [[ $(clang-format --version) != **"version 6"** ]]; then
+system=$(uname -s)
+
+if [[ $(clang-format --version) != *"version 6"* ]]; then
   echo "Please upgrade to clang-format version 6."
   echo "If it's installed via homebrew you can run: brew upgrade clang-format"
   exit 1
 fi
 
-if [[ $# -gt 0 && "$1" = "test-only" ]]; then
+if [[ "$system" == "Darwin" ]]; then
+  version=$(swiftformat --version)
+  version="${version/*version /}"
+  # Allow an older swiftformat because travis isn't running high sierra yet
+  # and the formula hasn't been updated in a while on sierra :-/.
```

Comment:
FYI, we can force High Sierra by changing to osx_image: xcode9.3beta. See https://docs.travis-ci.com/user/reference/osx/. Probably better not to run beta's on travis though.

Reply:
Agreed. Let's leave it for now.